Marlin/src/lcd/e3v2/proui/dwin.h

```diff
@@ -234,7 +234,7 @@ void doCoolDown();
 #if ENABLED(NOZZLE_PARK_FEATURE)
   void parkHead();
 #endif
-#if HAS_ONESTEP_LEVELING
+#if HAS_AUTOLEVEL
   void trammingwizard();
```

Comment:
Note that tramming is not leveling. And "one step" leveling is any `G29` procedure that can be completed without human intervention. So the tramming wizard should not be associated with `HAS_AUTOLEVEL`.

Reply:
![Screenshot 2023-12-08 040838](https://github.com/MarlinFirmware/Marlin/assets/18502096/7ade20eb-a764-4b6c-86d5-b7fb00826b2b)

Reply:
The "tramming wizard" may involve the probe, but it is a feature independent of leveling, and it should be possible to disable it even if a probe exists and leveling is enabled. These things are separate in the configurations, and I have been encouraging the authors of UI code to respect these settings in our discussions. We should continue to encourage and develop in the direction of properly applying configuration settings rather than perpetuating the oversights of UI authors.

Reply:
> It may help to bring @mriscoc into the conversation about changes to ProUI since some of these ProUI flags may be modified to use existing configuration options that apply to other UIs.

I currently use:
```cpp
#if ENABLED(LCD_BED_TRAMMING)
  void drawTrammingMenu();
  #if HAS_TRAMMING_WIZARD
    void runTrammingWizard();
  #endif
#endif
```